src/main/java/it/pn/frontend/e2e/pages/destinatario/personaGiuridica/DeleghePGPagoPAPage.java

```diff
@@ -332,6 +332,56 @@ public boolean siVisualizzaUnaDelegaPG() {
         }
     }
 
+    public void checkAlertWrongDelegationCode() {
+        try {
+            By alertWrongCode = By.id("alert-api-status}");
+            By alertCloseButtonBy = By.xpath("//*[@aria-label='Close']");
+            this.getWebDriverWait(10).withMessage("Alert non visualizzato correttamente").until(ExpectedConditions.visibilityOfElementLocated(alertWrongCode));
+            logger.info("Alert visualizzato correttamente in pagina");
+            WebElement alertCloseButton = driver.findElement(alertCloseButtonBy);
+            alertCloseButton.click();
+            logger.info("Alert chiusa");
+        } catch (TimeoutException e) {
+            logger.error("Alert non visualizzato con errore: " + e.getMessage());
+            Assert.fail("Alert non visualizzato con errore: " + e.getMessage());
+        }
+    }
+
+    public void clickButtonIndietroDaAssegnaGruppo() {
+        try {
+            logger.info("Si clicca sul bottone indietro per tornare al pop-up di inserimento codice delega");
+            By buttonIndietroPopUpAssegnaGruppo = By.xpath("//*[@data-testid='groupCancelButton']");
```

Comment:
Specify the correct element (instead of `*`) you want to select to prevent any potential issues in the future